docs/IDENTITIES.md

```diff
@@ -143,7 +143,7 @@ const verifyIdentity = identity => {
 }
 
 // The identity provider.
-const MyCustomIdentityProvider = ({ keystore }) => {
+const MyCustomIdentityProvider = ({ keystore }) => () => {
```

Comment:
Should add `async` here for the returned function to indicate that dev can do async io within the IP.